### PR TITLE
Feature/create banner component

### PR DESCRIPTION
**Pull request by Oliversw**
- Changed 'Home' page to 'Landing' to reflect our naming patterns in Figma
- Created Banner component with Logo for landing and error pages
- Created styled components file for Banner

😄 

#31

### CODE REVIEW

File: client/src/Component/views/Landing.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Banner from '../Common/Banner/index';
```
Comment (hshahwan) [inline client/src/Component/views/Landing.js:2]: I think you need to create a file called index inside Banner folder. I think you forget to raise it here.

**Review by hshahwan — CHANGES_REQUESTED** [review]

**Comment by hshahwan** [comment]
@Oliversw I think if correct the path of banner your Travis will work :), btw Good job :+1:

File: client/src/Component/views/Landing.js (hunk 1 above)
Comment (Oliversw) [reply client/src/Component/views/Landing.js:2]: This is so strange! I have an index file on my local branch and everything works. On my Github branch there is no index file, but there is one in this pull request just below this file. There does seem to be something weird with the folder capitalization in the one below though, so maybe that is having an effect ...

**Comment by hshahwan** [comment]
@Oliversw there are two folders for component delete the 'component' with the small letter

**Review by ali-7 — APPROVED** [review]
Good :))

**Pull request merged**